source/_docs/blueprint/tutorial.markdown

```diff
@@ -0,0 +1,220 @@
+---
+title: "Blueprint Tutorial"
+description: "Tutorial on creating a blueprint."
+---
+
+In this tutorial we're going to create a blueprint that controls a light based on a motion sensor. We will do this by taking an existing automation and converting it to a blueprint.
+
+For this tutorial we use a simple automation. The process for converting a complex automation is not any different.
+
+## Our automation
+
+To create a blueprint, we first need to have a working automation. The automation we're going to use in this tutorial, which controls a light based on a motion sensor, looks like this:
+
+{% raw %}
+```yaml
+trigger:
+  platform: state
+  entity_id: binary_sensor.motion_kitchen
+
+action:
+  service: >
+    {% if trigger.to_state.state == "on" %}
+      light.turn_on
+    {% else %}
+      light.turn_off
+    {% endif %}
+  target:
+    entity_id: light.kitchen
+```
+{% endraw %}
+
+## Create the blueprint file
+
+Automation blueprints are YAML files (with the `.yaml` extension) and live in the `<config>/blueprints/automation/` folder. You can create as many subdirectories in this folder as you want.
+
+To get started with our blueprint, we're going to copy the above automation YAML and save it in that directory with the name `motion_light_tutorial.yaml`.
+
+## Add basic blueprint metadata
+
+Home Assistant needs to know about the blueprint. This is achieved by adding a `blueprint:` section. It should contain the `domain` of the integration it is for (`automation`) and `name`, the name of your blueprint. Optionally, you can also include a `description` for your blueprint.
+
+Add this to the top of the file:
+
+```yaml
+blueprint:
+  name: Motion Light Tutorial
+  domain: automation
+```
+
+## Define the configurable parts as inputs
+
+Now we have to decide what steps we want to make configurable. We want to make it as re-usable as possible, without losing its original intent of turning on a light based on a motion sensor.
+
+Configurable parts in blueprints are called inputs. To make the motion sensor entity configurable, we're replacing the entity ID with a special YAML tag `!input`. This YAML tag has to be combined with the name of the input:
+
+```yaml
+trigger:
+  platform: state
+  entity_id: !input motion_sensor
+```
+
+For the light we can offer some more flexibility. We want to allow the user to be able to define any device or area as the target. The `target` property in the service action can contain references to areas, devices and/or entities, so that's what we will use.
```

Comment:
```suggestion
For the light, we can offer some more flexibility. We want to allow the user to be able to define any device or area as the target. The `target` property in the service action can contain references to areas, devices and/or entities, so that's what we will use.
```